### PR TITLE
[ARM64] Fix UMEntryThunkCode::Poison method

### PR DESCRIPTION
**Pull request by kbaladurin**
Fix hex value of the instruction ldp x16, x0, [x12]

### CODE REVIEW

**Comment by kbaladurin** [comment]
@jkotas @janvorli PTAL

**Comment by jkotas** [comment]
cc @sdmaclea

File: src/vm/arm64/stubs.cpp [hunk 1]
```diff
@@ -1275,7 +1275,7 @@ void UMEntryThunkCode::Poison()
     m_pTargetCode = (TADDR)UMEntryThunk::ReportViolation;
 
     // ldp x16, x0, [x12]
-    m_code[1] = 0xd42017c0;
+    m_code[1] = 0xa9400190;
```
Comment (sdmaclea) [inline src/vm/arm64/stubs.cpp]: New encoding matches the comment.
Previous encoding was `BRK #0xbe`

Comment (jkotas) [reply src/vm/arm64/stubs.cpp]: Thanks for checking!

**Comment by kbaladurin** [comment]
Thank you!

**Pull request merged**